.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1779,5 +1795,17 @@ dynamodb_Scenario_PartiQLBatch:
                 in batches.
               snippet_tags:
                 - python.example_code.dynamodb.Scenario_PartiQLBatch
+    C++:
+      versions:
+        - sdk_version: 1
+          github: cpp/example_code/dynamodb
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - cpp.example_code.dynamodb.Scenario_PartiQL_Batch
+                - cpp.example_code.dynamodb.scenario.createTable

```

Comment:
Same question, are these duplicate snippets intentional?

Reply:
Yes, but it will make sense when I add the code that references create and delete table,